### PR TITLE
Add rake tasks to support seeding.

### PR DESCRIPTION
**Pull request by justinlittman**
## Why was this change made? 🤔
Provide an approach to allow seeding APOs / Admin Policies for QA/stage reset.

## How was this change tested? 🤨

⚡ ⚠ If this change has cross service impact, including data writes to shared file systems, ***run [integration tests](https://github.com/sul-dlss/infrastructure-integration-test)*** and/or test in [stage|qa] environment, in addition to specs. ⚡

QA

### CODE REVIEW

**Review by mjgiarlo — COMMENTED** [review]
😍

**Comment by jcoyne** [comment]
The conventional place for this in a rails app is `db/seed.rb` Then when you call `bin/rake db:seed` it's loaded.

File: app/validators/cocina/apo_existence_validator.rb [hunk 1]
```diff
@@ -23,6 +26,6 @@ def valid?
 
     private
 
-    attr_reader :apo_id
+    attr_reader :apo_id, :druid
```
Comment (jcoyne) [inline app/validators/cocina/apo_existence_validator.rb:29]: Why is this change needed?

Comment (justinlittman) [reply app/validators/cocina/apo_existence_validator.rb]: It's not. Remnants of the code path not taken. Removed.

**Review by mjgiarlo — APPROVED** [review]
